### PR TITLE
Fix/kls 1042 image metadata

### PR DESCRIPTION
**Pull request by davidu1975**
Fix Image upload Alt Text, where Alt text was being set to the name of the file uploaded

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-1042
- [x] Jira ticket has the correct status.
- [x] A clear/description pull request messaged added.

### Housekeeping

- [x] I have checked that my PR is using the latest package versions of: great-components, directory-constants, directory-healthcheck, directory-validators, directory-components, directory-api-client, directory-ch-client, django-staff-sso-client, directory-forms-api-client, directory-sso-api-client, sigauth

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -989,6 +989,3 @@
     'SERVE_INCLUDE_SCHEMA': False,
     'PREPROCESSING_HOOKS': ['config.preprocessors.preprocessing_filter_admin_spec'],
 }
-
-# Data Workspace
-DATA_WORKSPACE_DATASETS_URL = env.str('DATA_WORKSPACE_DATASETS_URL', 'postgresql://')
```
Comment (sebs-code) [inline config/settings.py:994]: Is this ENV replaced anywhere? Or totally redundant now?

File: makefile [hunk 2]
```diff
@@ -32,7 +32,7 @@ ENV_FILES?='test,dev'
 pytest_codecov:
 	ENV_FILES=$(ENV_FILES) \
 	pytest \
-		tests/unit \
+		tests/unit/core \
```
Comment (sebs-code) [inline makefile:35]: Why only core tests?

File: core/views.py [hunk 3]
```diff
@@ -576,3 +585,43 @@ def form_valid(self, form):
 
 class GetExportHelpExperimentConfirmationView(TemplateView):
     template_name = 'core/get_export_help_experiment_confirmation.html'
+
+
+class AltImageUploadView(ImageUploadViewMixin, ImageCreationFormMixin, ImageChosenResponseMixin, View):
+    def post(self, request):
```
Comment (sebs-code) [inline core/views.py:591]: Would be good to have a Docstring here explaining why this class is needed (and how it relates to the generic Wagtail one).

File: config/settings.py (hunk 1 above)
Comment (davidu1975) [reply config/settings.py:994]: will have to ask Tim as he removed it

Comment (davidu1975) [reply config/settings.py:994]: yes was missed as part of data cleanup - okay to remove

**Review by sebs-code — APPROVED** [review]

**Pull request merged**